src/main/java/PanoViewer/gui/Menu.java

```diff
@@ -0,0 +1,36 @@
+package PanoViewer.gui;
+
+import javax.swing.*;
+
+public class Menu extends JMenuBar {
+   private JMenuBar menuBar;//making a menubar
```

Comment:
Why are you creating a extra JMenubar when your class itself is an extension of JMenubar?